src/test/java/uk/gov/justice/digital/delius/transformers/ConvictionTransformerTest.java

```diff
@@ -325,6 +328,59 @@ public void orderManagerAllocationReasonIsAlwaysNexEventCreated() {
 
     }
 
+    @Test
+    public void unpaidWorkMappedWherePresent() {
+        Event event = Event.builder()
+                .disposal(Disposal.builder()
+                        .unpaidWorkDetails(UpwDetails.builder()
+                                .upwLengthMinutes(120L)
+                                .appointments(Arrays.asList(
+                                        UpwAppointment.builder()
+                                                .attended("Y")
+                                                .complied("Y")
+                                                .minutesCredited(60L)
+                                        .build(),
+                                        UpwAppointment.builder()
+                                                .attended("N")
+                                                .complied("Y")
+                                                .build(),
+                                        UpwAppointment.builder()
+                                                .attended("N")
+                                                .complied("N")
+                                                .build(),
+                                        UpwAppointment.builder()
+                                                .attended(null)
+                                                .complied(null)
+                                                .build()
+                                        ))
+                                .build())
+                        .build())
+                .build();
+        UnpaidWork unpaidWork = transformer.convictionOf(event)
+                .getSentence()
+                .getUnpaidWork();
+
+        assertThat(unpaidWork).isNotNull();
+        assertThat(unpaidWork.getMinutesOrdered()).isEqualTo(120L);
+        assertThat(unpaidWork.getMinutesCompleted()).isEqualTo(60L);
```

Comment:
Think I'd like a test which summed the minutes across multiple UpwAppointments - if that is what getMinutesCompleted is doing